README.md

```diff
@@ -21,18 +21,29 @@ This is a collection of tasks executed with the Jake task runner. See http://jak
 Feel free to make additions to this repo.  
 If you are changing the namespace structure or an existing task please submit a PR.
 
-### Example Deploy Command
+## How to Deploy to Staging or Production
+Make sure your .env file is up to date 
+(Example: `PATH_TO_CONSOLE=/Users/bgreene/GreenChef/console-web/`)
+
+1. Navigate to the folder you want to deploy. (Example: `greenchef/services/server-greenchef`)
+2. Check out the branch you want to deploy. (Example: `release/2018-12-04`)
+3. View the `apps` constant in the `deploy.jake` file in `jakelib` to find the correct name of the docker image you want to push to
+(Example: `greenchef/services/server-greenchef` can be deployed to `consoleapi`, `web-api`, or `worker`)
+4. In a separate terminal tab, open the jakefiles repo, and run the deployment command (Example Below)
+
+#### Example Deployment Command
 Deploy the "console" app to the "staging-uat" ECS cluster.
+(Staging and Production cluster names can be found on AWS in ECS Clusters)
 ```bash
 jake deploy:app['staging-uat','console']
 ```
 
-### To Deploy Shipping to Staging UAT
-1. create a staging branch
+## How to Deploy the Shipping Platform to Staging
+1. create a staging branch (Example: `staging/name-of-current-sprint`)
 2. merge in branch `GCT-314-get-shipping-working` (this branch is not in prod)

```

Comment:
Is this step still needed?  I don't see a branch with that name in the shipping repo anymore.

Reply:
Yes, I checked with them.